src/web_app_skeleton/src/actions/api_action.cr.ecr

```diff
@@ -1,5 +1,6 @@
 # Include modules and add methods that are for all API requests
 abstract class ApiAction < Lucky::Action
+  disable_cookies
```

Comment:
Can we add a little comment on what this is removing? It removes sessions too, doesn't it?

Reply:
Added comment @jwoertink - feel free to change it to something better :)

Reply:
Thanks for catching this and adding it. It makes sense to disable cookies by default IMO! Is this right that "Cookie header will still be sent"? I thought the header wouldn't be sent if cookies are disabled?

Reply:
@paulcsmith you are right. I must have been confused of the request headers. I change the comment now.

Feel free to merge it if it looks good :)